File: backend/RestBook.Reservas/DTOs/CreateReservaDto.cs
namespace RestBook.Reservas.DTOs
{
    public class CreateReservaDto
    {
        public string NombreCliente { get; set; }
        public string CorreoCliente { get; set; }
        public DateTime FechaHora { get; set; }
        public int CantidadPersonas { get; set; }
        public int MesaId { get; set; }
    }
}

File: frontend/src/app/core/models/create-reserva-dto.ts
export interface CreateReservaDto {
  nombreCliente: string;
  correoCliente: string;
  fechaHora: string;       // p.ej. '2025-07-18T10:33:17.702Z'
  cantidadPersonas: number;
  mesaId: number;
}
